compiler/rustc_error_codes/src/error_codes/E0019.md

```diff
@@ -4,12 +4,14 @@ because the expression's value must be known at compile-time.
 Erroneous code example:
 
 ```compile_fail,E0019
```

Comment:
I guess we can nix this error code and use E0015 for asm, too

Reply:
How do we remove error codes? It's not documented in the dev-guide AFAICT.

Reply:
you remove all mentions of it from the source. There's no "unused old codes" list like with lints